scripts/core_uploader.sh

```diff
@@ -71,7 +69,6 @@ else
 
     # Zip the crashfiles
     zip -r $CRASH_FOLDER/${CORE_FILENAME}.all.zip $CRASH_FOLDER
-    mv /${CORE_FILENAME}.all.zip $CRASH_FOLDER/${CORE_FILENAME}.all.zip
```

Comment:
why is this removed?

Reply:
It looks like we're already zipping the file to the crash folder. This line just errors out and does nothing. Checked S3, the .all.zip file is uploaded to s3.